core/server/services/url/Urls.js

```diff
@@ -65,6 +65,26 @@ class Urls {
         }, []);
     }
 
+    /**
+     * @NOTE:
+     * It's is in theory possible that:
+     *
+     *  - resource1 -> /welcome/
+     *  - resource2 -> /welcome/
```

Comment:
@kirrg001 in the case described above will those two resources still be present in `this.urls` hash?

Reply:
Also we can use `getByResourceId` i think. Will simplify the target commit 👍